addon/constants/links.js

```diff
@@ -75,7 +75,7 @@ export default [{
     type: 'link'
   }, {
     href: 'https://help-wanted.emberjs.com/',
-    name: 'Contribute (Github)',
+    name: 'Contribute',
```

Comment:
What if we just literally say Help Wanted for now and when we have the contributing guides we can revise?

```suggestion
    name: 'Help Wanted',
```

Reply:
oooo I like that idea 🎉  @MelSumner how does that sound to you?